### PR TITLE
Mohosbalint/index

### PR DESCRIPTION
**Pull request by mohosb**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+			<div class="nav_btn">Home</div>
+            <div class="nav_btn">Skateboard</div>
+            <div class="nav_btn">Tricks</div>
+            <div class="nav_btn">Shop</div>
+            <div class="nav_btn">News</div>
+            <div class="nav_btn">Forum</div>
+            <div class="nav_btn">Contacts</div>
+		</nav>
+        <div id="help"></div>
+		<main></main>
```
Comment (tberes) [inline index.html]: Add an id to this e.g. < main id="main-content" >

File: style/css/desktop/site.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    top: 0;
+    left: 0;
+    right: 0;
+    height: 100px;
+    background-color: #252120;
+}
+
+#main-content {
+    background-image: url(../../images/site/background.png);
+    background-size: cover;
```
Comment (tberes) [inline style/css/desktop/site.css:12]: Add background-position-x: center;

File: style/css/mobile/site.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    top: 0;
+    left: 0;
+    right: 0;
+    height: 64px;
+    background-color: #252120;
+}
+
+#main-content {
+    background-image: url(../../images/site/background.png);
+    background-size: 250% 100%;
```
Comment (tberes) [inline style/css/mobile/site.css]: Use: background-size: cover please

File: style/css/global/main.css [hunk 4]
```diff
@@ -0,0 +1,8 @@
+/* Global */
+@import url("common.css");
+
+/* Mobile */
+@import url("../../css/mobile/site.css") only screen and (max-width: 480px);
```
Comment (tberes) [inline style/css/global/main.css]: The breakpoint should be 640px;

File: style/css/global/main.css [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+
+/* Desktop */
+@import url("../../css/desktop/site.css") only screen and (min-width: 480px);
```
Comment (tberes) [inline style/css/global/main.css]: The breakpoint should be 640px;

File: style/css/desktop/site.css (hunk 2 above)
Comment (mohosb) [reply style/css/desktop/site.css:12]: It doesn't change anything, besause the picture is already in the center.

**Pull request merged**